docs/src/content/docs/commands/ZPOPMIN.md

```diff
@@ -0,0 +1,137 @@
+- If the key does not exist, an empty array is returned.
+- If the key exists but is not a sorted set, an error is returned.
+- If the `count` argument is specified, up to that number of members with the lowest scores are returned and removed.
+- The returned array contains the members and their corresponding scores in the order of lowest to highest.
+
+## Errors
+1. `Wrong type error`:
+    - Error Message: `(error) WRONGTYPE Operation against a key holding the wrong kind of value`
+    - Occurs when trying to use the command on a key that is not a sorted set.
+
+2. `Syntax error`:
+    - Error Message: `(error) ERROR wrong number of arguments for 'zpopmin' command`
+    - Occurs when the command syntax is incorrect or missing required parameters.
+
+3. `Invalid argument type error`:
+    - Error Message : `(error) ERR value is not an integer or out of range`
+    - Occurs when the count argument passed to the command is not an integer.
+
+## Examples
+
+### Non-Existing Key (without count argument)
+
+Attempting to pop the member with the lowest score from a non-existent sorted set:
+
+```bash
+127.0.0.1:7379> ZPOPMIN NON_EXISTENT_KEY
+(empty array)
+```
+
+### Wrong Type of Key (without count argument)
+
+Attempting to pop from a key that is not a sorted set:
+
+```bash
+127.0.0.1:7379> SET stringkey "string_value"
+OK
+127.0.0.1:7379> ZPOPMIN stringkey
+(error) WRONGTYPE Operation against a key holding the wrong kind of value
+```
+
+### Existing Key (without count argument)
+
+Popping the member with the lowest score from an existing sorted set:
+
+```bash
+127.0.0.1:7379> ZADD myzset 1 member1 2 member2 3 member3
+(integer) 3
+127.0.0.1:7379> ZPOPMIN myzset
+1) 1 "member1"
+```
+
+### With Count Argument
+
+Popping multiple members with the lowest scores using the count argument:
+
+```bash
+127.0.0.1:7379> ZADD myzset 1 member1 2 member2 3 member3
+(integer) 3
+127.0.0.1:7379> ZPOPMIN myzset 2
+1) 1 "member1"
+2) 2 "member2"
+```
+
+### Count Argument but Multiple Members Have the Same Score
+
+Popping members when multiple members share the same score:
+
+```bash
+127.0.0.1:7379> ZADD myzset 1 member1 1 member2 1 member3
+(integer) 3
+127.0.0.1:7379> ZPOPMIN myzset 2
+1) 1 "member1"
+2) 1 "member2"
+```
+
+### Negative Count Argument
+
+Attempting to pop members using a negative count argument:
+
+```bash
+127.0.0.1:7379> ZADD myzset 1 member1 2 member2 3 member3
+(integer) 3
+127.0.0.1:7379> ZPOPMIN myzset -1
+(empty array)
+```
+
+### Invalid Count Argument
+
+Using an invalid (non-integer) count argument:
+
+```bash
+127.0.0.1:7379> ZADD myzset 1 member1
+(integer) 1
+127.0.0.1:7379> ZPOPMIN myzset INCORRECT_COUNT_ARGUMENT
+(error) ERR value is not an integer or out of range
+```
+
+### Floating-Point Scores
+
+Popping members with floating-point scores:
+
+```bash
+127.0.0.1:7379> ZADD myzset 1.5 member1 2.7 member2 3.8 member3
+(integer) 3
+127.0.0.1:7379> ZPOPMIN myzset
+1) 1.5 "member1"
+```
```

Comment:
nit: Happy path examples should come before the examples for error paths.

Reply:
Yea that's right! I will make these changes

Reply:
Made the changes in this commit: [773521b](https://github.com/DiceDB/dice/pull/1143/commits/773521b261b16242553ac63b4a185801dca5cb87)